src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,268 @@
+Runge-Kutta-Dormand-Prince uses adaptive step size, which can be clipped by
+this value.
+
+
+Tolerances
+""""""""""
+
+RKDP, Adams-Bashforth, and MD3-C1 make use of the tolerance options.
+    
+RKDP
+    Two tolerances are available - The adaptive step is set such that the
+    maximum error at each step is less than the maximum between the absolute
+    tolerance and the relative tolerance times the value of the vector field at
+    the current point. The absolute tolerance can be truly absolute or relative
+    to the bounding box.
+
+Adams-Bashforth
+    NOT REALLY CLEAR WHAT THIS IS BECAUSE AB USES A FIXED-STEP SIZE. MAYBE THIS
+    IS A CLIP?
+
+MD3-C1
+    NOT REALLY CLEAR WHAT THIS IS BECAUSE AB USES A FIXED-STEP SIZE. MAYBE THIS
+    IS A CLIP?
+
+
+Termination
+"""""""""""
+
+The criteria for terminating the integration. See specific operator for
+details.
+
+
+
+Appearance
+^^^^^^^^^^
+
+Specify appearance settings for the curves. See specific operator for details
+on most settings.
+
+Streamlines vs Pathlines
+""""""""""""""""""""""""
+
+The user may select the integral curve to be based on an instantaneous or
+time-varying vector field producing streamlines or pathlines, respectively. A
+streamline is path drawn by an integrator that uses the same vector field for
+the entire integration. A pathline uses the vector field that is in-step with
+the integrator, so as the integrator steps through time, it draws from the
+vector field at that new time step.
+    
+Pathline Options:
+    
+    * Override starting time - instead of starting with the current time step,
+    utilize another time for the start time.
+   
+       * Time - new starting time.
+      
+    * Interpolation over time - indicated whether the mesh changes over time. 
+   
+       * Mesh is static over time - typically the case but can not be assumed.
+         Confirm this before selecting.
+       * Mesh varies over time (Default).
+
+Advanced
+^^^^^^^^
+
+Parallel integration
+""""""""""""""""""""
+
+The user may select one of four different parallelization options when
+integrating curves in parallel:
+
+Parallelize over curves
+    Distribute the curves between the processors. 
+
+    * Domain cache size - number of blocks to hold in memory for level of
+      details (LOD). 
+   
+Parallelize over domains
+    Distribute the domains between the processors. 
+
+    * Communication threshold - number of integral curve to process before
+      communication occurs. 
+   
+Parallelize over curves and domains
+    Distribute the curves and domains between the processors.
+ 
+    * Communication threshold - number of integral curve to process before
+      communication occurs. 
+    * Domain cache size - number of blocks to hold in memory for level of
+      details (LOD). 
+    * Work group size - master slave also (currently disabled). 
+   
+Have VisIt select the best algorithm
+    VisIt automagically selects the best parallelization algorithm.
+
```

Comment:
Is there any user-level giudance on why/when choosing one over the other options makes sense here?

Reply:
Not that I saw in the original docs. I will have to dig under the hood a bit to see how VisIt chooses the best algorithm.

Reply:
@dpugmire is the best person to answer this question